sys-cluster/keepalived/keepalived-2.2.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools systemd
+
+DESCRIPTION="A strong & robust keepalive facility to the Linux Virtual Server project"
+HOMEPAGE="https://www.keepalived.org/"
+SRC_URI="https://www.keepalived.org/software/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="+bfd dbus -json regex snmp systemd"
+
+RDEPEND="dev-libs/libnl:=
```

Comment:
We need to pull in systemd for USE=systemd like it is required at build time, https://github.com/acassen/keepalived/blob/v2.2.1/keepalived/core/Makefile.am#L47-L50

Reply:
I thought users of systemd will implicitly have this set and nobody with openrc will just set USE=systemd. But yes, you're right, I'll make it explicit.